samples/core/kfp_env_validation/kfp_env_validation.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>In line 11 perhaps add a comment regarding why we use this workaround (because the image we're going to use has no pip installed).</p><p> </p><p>In line 16 shall we bind the kfp version?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/2843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Binding the KFP version to 'kfp>=0.1.31' ,  also additional details to comment  before pipe install as follows

# Installing pip3 and kfp, since the base image 'google/cloud-sdk:latest' does not come with pip3 pre-installed.

LMK how that sounds.